source/AMDSClient.cpp

```diff
@@ -344,6 +345,20 @@ void AMDSClient::requestData()
 			clientMiddleTimePlusCountBeforeAndAfterDataRequest->setCountAfter(countAfter);
 		}
 	}
+	else if(clientRequest->requestType() == AMDSClientRequestDefinitions::Continuous){
+		AMDSClientContinuousDataRequest *clientContinuousDataRequest = qobject_cast<AMDSClientContinuousDataRequest*>(clientRequest);
+		if(clientContinuousDataRequest){
+			quint64 updateInterval = 500;
```

Comment:
@dretrex We took 500ms as the default update interval in case the user forget to provide one.

Yes, we don't need quint64. quint32 should be more than enough.